### PR TITLE
bug fix: Console Error when open Indicator Setting

### PR DESCRIPTION
**Pull request by brucebinary**
the error was caused by Slider.jsx because newer versions of prop types do not use the legacy getDefaultProps declaration anymore. The solution here rewrites Slider.jsx to use modern React code instead of legacy createClass to solve the issue.

This only happens in production versions of the library, and frankly aside the console error no harm is done.

### CODE REVIEW

File: sass/components/_ciq-form.scss [hunk 1]
```diff
@@ -8,8 +8,8 @@
     .slider {
         width: 120px;
         height: 10px;
-        display: relative;
         cursor: pointer;
+        touch-action: manipulation;
```
Comment (brucebinary) [inline sass/components/_ciq-form.scss]: just added this to stop `Intervention] Unable to preventDefault inside passive event listener due to target being treated as passive.` error message. Refer https://github.com/OwlCarousel2/OwlCarousel2/issues/1932

File: yarn.lock [hunk 2]
```diff
@@ -2329,14 +2329,6 @@ create-hmac@^1.1.0, create-hmac@^1.1.2, create-hmac@^1.1.4:
     safe-buffer "^5.0.1"
     sha.js "^2.4.8"
 
-create-react-class@^15.6.3:
```
[the remark this answers is not in the document]
Comment (brucebinary) [reply yarn.lock]: In fact you should do the same if you add/remove packages

[the remark this answers is not in the document]
Comment (shayanbinary) [reply yarn.lock]: @brucebinary I keep this in mind because for now,  I ignore this file on my commits.

**Comment by shayanbinary** [comment]
LGTM

**Pull request merged**